lcUILua/ui/commandline.lua

```diff
@@ -94,8 +104,8 @@ end
 
 --Register every commands
 add_command("LINE", function(id) run_basic_operation(id, LineOperations) end)
-add_command("CIRCLE", function(id) run_basic_operation(id, CircleOperations) end)
-add_command("ARC", function(id) run_basic_operation(id, ArcOperations) end)
+add_command("CIRCLE", function(id) op = run_basic_operation(id, CircleOperations); op:_init_default()  end)
```

Comment:
`_init_default` means that it should be called by default.
`run_basic_operation` is a helper to prevent us repeating ourself every time in commands or buttons. You should add a third argument that contains the name of the method to run.

I see 2 cases :
1. You have the name of init method to run, you can call a method dynamically like this : https://github.com/LibreCAD/LibreCAD_3/blob/e48d60e77fa7008ad8ce849569f153978f47c529/lcUILua/createActions/createOperations.lua#L26

2. You don't have the name of the method (`method == nil`). You have to run `_init_default()` if it's present.